src/lavinmq/queue/queue_factory.cr

```diff
@@ -52,5 +54,11 @@ module LavinMQ
     private def self.stream_queue?(frame) : Bool
       frame.arguments["x-queue-type"]? == "stream"
     end
+
+    private def self.check_unsupported_queue_type(frame)
+      if frame.arguments["x-queue-type"]?
+        Log.warn { "The queue type #{frame.arguments["x-queue-type"]} is not supported by LavinMQ and will revert to default" }
```

Comment:
Maybe this is a little more clear? Just a suggestion though!
```suggestion
        Log.warn { "The queue type #{frame.arguments["x-queue-type"]} is not supported by LavinMQ and will be changed to the default queue type" }
```

Reply:
nice!